src/config.rs

```diff
@@ -235,6 +235,11 @@ pub struct SharedConfigValues {
     #[structopt(long, parse(from_os_str), short, value_name = "string")]
     cache_path: Option<PathBuf>,
 
+    /// Enable to use audio cache
+    #[structopt(long)]
+    #[serde(default, deserialize_with = "de_from_str")]
+    audio_cache: bool,
```

Comment:
I think I would rename it to cache_audio. Makes the command line more “readable”:
`spotifyd —no-daemon —cache-audio —use-keyring`

It also matches more the naming scheme and makes reading the line more natural.

Reply:
Oh wait nevermind. If we use true as the default the flag should probably be no-audio-cache instead. My bad!